pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -83,6 +84,58 @@ type cdcTester struct {
 
 	workloadWg *sync.WaitGroup
 	doneCh     chan struct{}
+
+	// statsExporter is used to generate a
+	// statistics file for roachperf to consume.
+	statsExporter *cdcStatsExporter
```

Comment:
What do you think about not even storing a `cdcStatsExporter` struct at all by just having
```go
func (ct *cdcTester) startStatsCollection() func() {
  startTime = timeutil.Now()
  return func() { // this returned function is then called in the test to export to roachperf
    	endTime := timeutil.Now()
	     ...ct.statsExporter.statsCollector.Exporter().Export(...)...
  }
}
```

Reply:
I like this better than what I had, but the reason I stored the stats exporter was because it gets created in `newCDCTester > startGrafana`. I just pushed a small refactor. Let me know if this is better.

Reply:
Does it need to be created there though rather than just in the function itself?  Grafana can be assumed to always be running since we only don't start it when `SkipInit` is set since the earlier instance should still be running.  I feel like its worth just commenting on the function that this requires prometheus.  

If it does need to be at the very beginning, I'd rather just have it be a property in `cdcTester` rather than an extra return value.

Reply:
I created it in `startGrafana` because we had the prometheus config in scope there. It is weird to initialize it there though. I would also prefer doing it in `startStatsCollection`. The only issue is that we have to store the `promCfg` in the `cdcTester`, which I think is fine.